src/main/java/org/terasology/dynamicCities/construction/TreeRemovalSystem.java

```diff
@@ -93,8 +93,8 @@ public boolean removeTreesInRegion(EntityRef region, Rect2i area) {
         TreeFacetComponent trees = region.getComponent(TreeFacetComponent.class);
         LocationComponent loc = region.getComponent(LocationComponent.class);
         Rect2i relevantArea = area.expand(SettlementConstants.MAX_TREE_RADIUS, SettlementConstants.MAX_TREE_RADIUS);
-        Region3i treeRegion = Region3i.createFromMinAndSize(new Vector3i(relevantArea.minX(), loc.getLocalPosition().y(), relevantArea.minY()),
-                new Vector3i(relevantArea.sizeX(), 32, relevantArea.sizeY()));
+        BlockRegion treeRegion = new BlockRegion(relevantArea.minX(), (int) loc.getLocalPosition().y(), relevantArea.minY(),
+                relevantArea.sizeX(), 32, relevantArea.sizeY());
```

Comment:
This was `createFromMinAndSize` before:
```suggestion
        BlockRegion treeRegion = new BlockRegion(relevantArea.minX(), (int) loc.getLocalPosition().y(), relevantArea.minY())
                .setSize(relevantArea.sizeX(), 32, relevantArea.sizeY());
```